tests/Teams/TeamTest.php

```diff
@@ -39,7 +38,7 @@ public function testHydrate()
 
     public function testHydrateWithoutCreatedAt()
     {
-        $team = new User();
+        $team = new Team();
```

Comment:
This test was likely copy pasted from the user test but we forgot to change the class here.

Reply:
Great catch!